typings/index.d.ts

```diff
@@ -1053,6 +1055,7 @@ export class GuildMember extends PartialTextBasedChannel(Base) {
   public createDM(force?: boolean): Promise<DMChannel>;
   public deleteDM(): Promise<DMChannel>;
   public displayAvatarURL(options?: ImageURLOptions): string;
+  public disableCommunication(duration: number, reason?: string): Promise<GuildMember>;
```

Comment:
The GuildMemberEditData interface still needs to be updated
```suggestion
  public disableCommunication(seconds: number, reason?: string): Promise<GuildMember>;
```